documentation/docs/libraries/rust/how_to/_06_get_address_balances/_key_api_ref_links.md

```diff
@@ -0,0 +1,33 @@
+### `Client.basic_output_ids(query_parameters)`
+
+You can retrieve outputs from any given address using the  
+the [`Client.basic_output_ids(query_parameters)`](iota_client/client/struct.Client.html#method.basic_output_ids)
```

Comment:
It's a bit weird that you put either a parameter name or both the name and the type

Reply:
This hasn't been handled